packages/spec/helpers/test-helpers.js

```diff
@@ -13,60 +14,67 @@ const resolveFrom = require('resolve-from');
 const build = async ({ cwd, env = {}, argv = [] }) => {
   const hopsBin = resolveFrom(cwd, 'hops/bin');
   const command = `${process.argv[0]} ${hopsBin} build ${argv.join(' ')}`;
+  const returnValue = { stdout: '', stderr: '' };
   debug('Starting', command);
   try {
-    return await exec(command, { env, cwd });
-  } catch (err) {
-    return { stderr: err.message };
+    const result = await exec(command, { env, cwd });
+    Object.assign(returnValue, result);
+  } catch (error) {
+    Object.assign(returnValue, { error });
```

Comment:
Why do we return different objects here?
I think we should always return `{ stdout: string, stderr: string, error: null | Error }` right?
Because now we need to get `stderr` through `error.message`, which could be made simpler: https://github.com/xing/hops/pull/1481/files#diff-85085ff19f5129dd6e046735baf4dbd5949625be39a9956e7a33b936f0454908R19

Reply:
`stderr` and `error` are different things. Even if we decide to always have an `error`-property on the resolved object, which defaults to `null`. That's what I meant with…

> align HopsCLI.build-API with child_process.exec-API